www/js/chart-api-test.js

```diff
@@ -26,7 +27,7 @@ function generateColorPattern(patternAreas) {
 
 
 function processApiMempoolDataForChart(response) {
-  console.log(response.timestamp)
+  console.log('Processed data at timestamp: ', response.timestamp)
```

Comment:
fyi this is more a timestamp of "time at writing to database" (no action needed here since it's only for dev reasons)